resources/views/play/show.blade.php

```diff
@@ -161,7 +161,11 @@
                         <th>公演日時</th>
                         <td rowspan="2">
                             @foreach($play[$ps]['lily:showTime'] as $showTime)
-                                <div style="min-width: 220px;">{{ \Carbon\Carbon::make($showTime)->isoFormat('YYYY年M月D日 (ddd) HH:mm') }}</div>
+                                @if(in_array($showTime, $play[$ps]['lily:cancelledShowTime'] ?? [], true))
+                                    <div style="min-width: 220px;"><s>{{ \Carbon\Carbon::make($showTime)->isoFormat('YYYY年M月D日 (ddd) HH:mm') }}</s> 中止</div>
```

Comment:
できればdel要素に置き換えてください 文書構造としてより適切になります

参考 : https://developer.mozilla.org/ja/docs/Web/HTML/Element/del

```suggestion
                                    <div style="min-width: 220px;"><del>{{ \Carbon\Carbon::make($showTime)->isoFormat('YYYY年M月D日 (ddd) HH:mm') }}</del> 中止</div>
```

Reply:
それも考えたんですが、「削除された文字列」かと言われたらそれも微妙じゃないですか？

Reply:
むしろ取り消し線自体をやめて、cssで文字色を薄めにしてかつ「中止」を見栄えの良いアイコンラベルみたいな感じにするのがいい気がしました

Reply:
確かに、日程の削除というよりは現時点で適切でないという文脈になるのでdelはむしろ不適切ですね。取り消してこれでMergeします。

Reply:
> むしろ取り消し線自体をやめて、cssで文字色を薄めにしてかつ「中止」を見栄えの良いアイコンラベルみたいな感じにするのがいい気がしました

良いと思います。後ほど考えてみますね。